diff --git a/App/ui/menu.c b/App/ui/menu.c
index 46e57e3..9d40038 100644
--- a/App/ui/menu.c
+++ b/App/ui/menu.c
@@ -131,6 +131,8 @@ const t_menu_item MenuList[] =
         {"AM Fix",      MENU_AM_FIX        },
     #endif
 #endif
+
+    {"SelCal",     MENU_SELECTIVE   },
     {"VOX",         MENU_VOX           },
 #ifdef ENABLE_FEAT_F4HWN
     {"SysInf",      MENU_VOL           }, // was "VOL"
@@ -202,6 +204,17 @@ const char gSubMenu_TXP[][6] =
     "HIGH"
 };
 
+const char * const gSubMenu_SELCAL[] =
+{
+    "OFF",
+    "ZVEI-1",
+    "ZVEI-2",
+    "CCIR",
+    "REGA TEST",
+    "REGA ALARM"
+};
+
+
 const char gSubMenu_SFT_D[][4] =
 {
     "OFF",
@@ -675,6 +688,11 @@ void UI_DisplayMenu(void)
             break;
         }
 
+        case MENU_SELECTIVE:
+            strcpy(String, gSubMenu_SELCAL[gSubMenuSelection]);
+            break;
+
+
         case MENU_SFT_D:
             strcpy(String, gSubMenu_SFT_D[gSubMenuSelection]);
             break;
